src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -505,11 +548,21 @@ public Response corsPreflightDelete(@HeaderParam("Access-Control-Request-Headers
 	@POST
 	@Produces(MediaType.APPLICATION_JSON + "; charset=UTF-8")
 	@Path("/datapoints/delete")
-	public Response delete(String json) throws Exception
+	public Response delete(@Context HttpHeaders httpHeaders, String json) throws Exception
 	{
 		checkNotNull(json);
 		logger.debug(json);
 
+		ScopeManager scopeManager = tracer.scopeManager();
+		SpanContext spanContext = tracer.extract(Format.Builtin.HTTP_HEADERS, new HttpHeadersCarrier(httpHeaders.getRequestHeaders()));
+		Tracer.SpanBuilder spanBuild = tracer.buildSpan("/datapoints/delete").withTag("Name","/datapoints/delete").withTag(Tags.SPAN_KIND.getKey(),Tags.SPAN_KIND_SERVER);
```

Comment:
Duplicate ``/datapoints/delete``

Reply:
Operation name should be snake_case: ``datapoints_delete``, ``datapoints_query`` etc...